core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -376,7 +376,7 @@ RemoteStorageManager createRemoteStorageManager() {
 
     private void configureRSM() {
         final Map<String, Object> rsmProps = new HashMap<>(rlmConfig.remoteStorageManagerProps());
-        rsmProps.put(ServerConfigs.BROKER_ID_CONFIG, brokerId);
+        rsmProps.put(KRaftConfigs.NODE_ID_CONFIG, brokerId);
```

Comment:
remote storage is using `broker.id` rather than `node.id`. see https://github.com/apache/kafka/blob/78e35457ee966695bd392aa4e4c16cbfe919ad4f/storage/src/test/java/org/apache/kafka/server/log/remote/storage/LocalTieredStorage.java#L238

maybe we should align the naming for this internal config. @showuon FYI

Reply:
Hi @chia7712, I updated `BROKER_ID` to `NODE_ID` to fix CI error. Could you take a look when you have time? Thank you.